js/controller/ContentController.js

```diff
@@ -8,8 +8,7 @@
  * @copyright Bernhard Posselt 2014
  */
 app.controller('ContentController', function (Publisher, FeedResource, ItemResource, SettingsResource, data, $route,
-                                              $routeParams, $location, FEED_TYPE, ITEM_AUTO_PAGE_SIZE, Loading,
-                                              $filter) {
```

Comment:
The `$filter` variable isn't used anymore, so the linter complains if it is still defined here.